packages/cloud-cognitive/src/components/CreateFullPage/CreateFullPage.js

```diff
@@ -0,0 +1,388 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Import portions of React that are needed.
+import React, { useEffect, useCallback, useState } from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { pkg } from '../../settings';
+import { CREATE_FULL_PAGE_STEP } from './constants';
+import uuidv4 from '../../global/js/utils/uuidv4';
+
+// Carbon and package components we use.
+import {
+  Grid,
+  ModalFooter,
+  ProgressIndicator,
+  ProgressStep,
+  Toggle,
+  ComposedModal,
+  ModalHeader,
+  ModalBody,
+  Button,
+} from 'carbon-components-react';
+import { ActionSet } from '../ActionSet';
+const blockClass = `${pkg.prefix}--create-full-page`;
+const componentName = 'CreateFullPage';
+
+export let CreateFullPage = React.forwardRef(
+  (
+    {
+      backButtonText,
+      cancelButtonText,
+      children,
+      className,
+      hasToggle,
+      modalDangerButtonText,
+      modalDescription,
+      modalSecondaryButtonText,
+      modalTitle,
+      nextButtonText,
+      onClose,
+      onRequestSubmit,
+      submitButtonText,
+      toggleAriaLabel,
+      toggleLabelText,
+      ...rest
+    },
+    ref
+  ) => {
+    const [createFullPageActions, setCreateFullPageActions] = useState([]);
+    const [currentStep, setCurrentStep] = useState(1);
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    const [open, setOpen] = useState(false);
```

Comment:
This value should most likely be in the component that uses `<CreateFullPage />`, and passed as a prop instead of being set within this component.

Reply:
Again, this is referring to the Modal inside of the CreateFullPage component. Here's a screenshot. 

<img width="966" alt="Screen Shot 2021-06-11 at 8 32 15 AM" src="https://user-images.githubusercontent.com/47571589/121694338-a4dccd00-ca8f-11eb-8203-1c8a439ee22b.png">

Reply:
Oh ok, that value should probably be renamed to `modalIsOpen` or something a bit more specific.